ContextMenu/ContextMenu+HapticFeedbackStyle.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ContextMenu+HapticFeedbackStyle.swift
+//  ContextMenu
+//
+//  Created by Abdurahim Jauzee on 31/12/2018.
+//
+
+import Foundation
+
+public extension ContextMenu {
+
+  /// Haptic Feedback types.
+  public enum ContextMenuHapticFeedbackStyle: Int {
```

Comment:
Can this be renamed to just `HapticFeedbackStyle` since it's already namespaces by the `ContextMenu` extension? That'd be consistent w/ the rest of the library.

Reply:
Yes of course. Done.